jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,25 +1,30 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: instrument_name.schema.yaml
+- $ref: detector.schema.yaml
+- $ref: filter_core.schema.yaml
```

Comment:
If there are going to be schemas for individual keywords, I would prefer longer and clearer names (such as filter_keyword), because these are in a category of their own, in some sense, and a common suffix would help us to find these schemas.